app/models/manageiq/providers/ibm_power_hmc/inventory/parser/infra_manager.rb

```diff
@@ -335,6 +336,20 @@ def parse_vm_guest_devices(lpar, hardware)
     lpar.lhea_ports.each do |ent|
       build_ethernet_dev(ent, hardware, "host ethernet adapter")
     end
+
+    # Physical adapters can be assigned to VIOSes and LPARs using IOMMU.
+    lpar.io_adapters.each do |io|
+      persister.guest_devices.build(
+        :hardware        => hardware,
+        :uid_ems         => io.dr_name,
+        :device_type     => "physical_port",
+        :controller_type => "IO",
+        :device_name     => "Adapter",
```

Comment:
The `device_name` is usually something unique to each device, is `io.dr_name` something that could be added here like `"Adapter #{io.dr_name}"` ?

Reply:
Yes. The reason I did this is that it does not render very well in the UI with the same information being displayed twice since we are also using the DR name as the location (it's the slot on the bus).

![image](https://user-images.githubusercontent.com/48122102/195018537-bb604893-5aa3-4c18-85cb-93a5687e9908.png)

I'm ok either way.

Reply:
Ah okay, I'm fine with this then